.github/workflows/build.yml

```diff
@@ -443,12 +443,104 @@ jobs:
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/distribution/ci APPLE_CERTIFICATE;
+            secret/data/products/distribution/ci APPLE_CERTIFICATE_PASSWORD;
+            secret/data/products/distribution/ci APPLE_DEVELOPER_ID;
+            secret/data/products/distribution/ci APPLE_DEVELOPER_PASSWORD;
+            secret/data/products/distribution/ci APPLE_TEAM_ID;
+            secret/data/products/distribution/ci APPLE_COMMON_NAME;
+
+      - name: Download MacOS Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: unsigned-native-darwin-amd64
+          path: artifacts/darwin-amd64
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: unsigned-native-darwin-aarch64
+          path: artifacts/darwin-aarch64
+
+      - name: Unzip MacOS Artifacts and save zip file name for later
+        id: unzip_artifacts
+        run: |
+          original_amd64_zip=$(ls artifacts/darwin-amd64/rpa-worker*.zip)
+          echo "amd64_zip_name=${original_amd64_zip##*/}" >> $GITHUB_ENV
+          unzip $original_amd64_zip -d artifacts/darwin-amd64/unziped
+
+          original_aarch64_zip=$(ls artifacts/darwin-aarch64/rpa-worker*.zip)
+          echo "aarch64_zip_name=${original_aarch64_zip##*/}" >> $GITHUB_ENV
+          unzip $original_aarch64_zip -d artifacts/darwin-aarch64/unziped
+
+      - name: Sign and notarize amd64
+        uses: ./.github/actions/sign-and-notarize
+        with:
+          p12-base64: ${{ steps.secrets.outputs.APPLE_CERTIFICATE }}
+          p12-password: ${{ steps.secrets.outputs.APPLE_CERTIFICATE_PASSWORD }}
+          developer-id-cert-name: ${{ steps.secrets.outputs.APPLE_DEVELOPER_ID }}
+          apple-id: ${{ steps.secrets.outputs.APPLE_DEVELOPER_ID }}
+          app-password: ${{ steps.secrets.outputs.APPLE_DEVELOPER_PASSWORD }}
+          team-id: ${{ steps.secrets.outputs.APPLE_TEAM_ID }}
+          path: artifacts/darwin-amd64/unziped
+
+      - name: Sign and notarize aarch64
+        uses: ./.github/actions/sign-and-notarize
+        with:
+          p12-base64: ${{ steps.secrets.outputs.APPLE_CERTIFICATE }}
+          p12-password: ${{ steps.secrets.outputs.APPLE_CERTIFICATE_PASSWORD }}
+          developer-id-cert-name: ${{ steps.secrets.outputs.APPLE_DEVELOPER_ID }}
+          apple-id: ${{ steps.secrets.outputs.APPLE_DEVELOPER_ID }}
+          app-password: ${{ steps.secrets.outputs.APPLE_DEVELOPER_PASSWORD }}
+          team-id: ${{ steps.secrets.outputs.APPLE_TEAM_ID }}
+          path: artifacts/darwin-aarch64/unziped
+
+      - name: Rezip Signed MacOS Artifacts
+        run: |
+          zip -r artifacts/${{ env.amd64_zip_name }} artifacts/darwin-amd64/unziped/*
+          zip -r artifacts/${{ env.aarch64_zip_name }} artifacts/darwin-aarch64/unziped/*
```

Comment:
This directory structure is being reflected in the zip:
```
  Length      Date    Time    Name
---------  ---------- -----   ----
     1518  2025-04-11 09:55   artifacts/darwin-aarch64/unziped/application.properties
146315152  2025-04-11 10:08   artifacts/darwin-aarch64/unziped/rpa-worker_1.0.2-test.1_darwin_aarch64
---------                     -------
146316670                     2 files
```
I think we should keep them at the root the same as all the others

Reply:
Good catch. I will adjust this